crates/webidl/src/generator.rs

```diff
@@ -445,8 +445,10 @@ impl InterfaceMethod<'_> {
                     let js_name = raw_ident(js_name);
                     extra_args.push(quote!( js_name = #js_name ));
                 }
+                let method_buffer;
                 let method = if *is_static {
-                    &format!("{js_name}_static")
+                    method_buffer = format!("{js_name}_static");
+                    &method_buffer
```

Comment:
I assume this was an issue because you are using an older version of Rust?
Will go ahead and revert this.